### PR TITLE
Create index.html

### PR DESCRIPTION
**Pull request by ETwilight**
This is the todolist written by Twilight

### CODE REVIEW

**Review by yechs — COMMENTED** [review]
Good developing practices: make your PR title and commit messages more meaningful

File: index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	
+	//delete by the identity
+	function del(id){
+		tdlist[id]=false;
+		update();
+	}
+
+  </script>
+ </body>
+</html>
```
Comment (yechs) [inline index.html:82]: Format: Add newline at end of file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  <meta name="Generator" content="EditPlus®">
+  <meta name="Author" content="Twilight">
+  <title>Todolist</title>
+ </head>
+
+ <body>
+  <h3>·Todolist</h3>
+  <h4>·Append:</h4>
+  <form action="">
+	&nbsp;&nbsp;&nbsp;&nbsp;<input id="add_todolist"></input>
```
Comment (yechs) [inline index.html:16]: Format: Use **consistent** style for indenting

i.e. don't mix spaces with tabs; and use the same amount of spaces for each indentatation

**Comment by yechs** [comment]
@feizaile 尽快把 coding style 的文档写好吧

**Review by madaomax — APPROVED** [review]

**Pull request merged**